doc/source/examples/quickstart.ipynb

```diff
@@ -1,28 +1,29 @@
 {
```

Comment:
<p>Hmm, it might be a good idea to sort this output so it doesn't change over each iteration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Fixed.</p>